framework/wazuh/core/cluster/common.py

```diff
@@ -125,7 +129,7 @@ class ReceiveStringTask:
     Create an asyncio task that can be identified by a task_id specified in advance.
     """
 
-    def __init__(self, wazuh_common, logger, task_id):
+    def __init__(self, wazuh_common, logger, task_id, info_type='Agent-info sync', payload=None):
```

Comment:
The default value for `info_type` (`'Agent-info sync'`) is never used. In `master.py`, `agent-info` and `agent-groups` are used instead. Maybe it shouldn't have default value or use one of these:
```PYTHON
        if info_type == 'agent-info':
            ...
        elif info_type == 'agent-groups':
            ...
```

Also, the `payload` parameter doesn't seem to be used anywhere.